backend/baseclass.pm

```diff
@@ -317,7 +317,7 @@ sub start_encoder ($self) {
     my $cwd = Cwd::getcwd;
     my @cmd = (qw(nice -n 19), "$bmwqemu::scriptdir/videoencoder", "$cwd/video.ogv");
     push(@cmd, '-n') if $bmwqemu::vars{NOVIDEO} || ($has_external_video_encoder_configured && !$bmwqemu::vars{EXTERNAL_VIDEO_ENCODER_ADDITIONALLY});
-    push(@cmd, "-x $self->{xres} -y $self->{yres}");
+    push @cmd, '-x', $self->{xres}, '-y', $self->{yres};
```

Comment:
I'm wondering why there is no failing test but also no warning about missing coverage 🤔

Reply:
> I'm wondering why there is no failing test

The line is surly covered by tests. I assume no test actually asserts whether the video resolution is correct (at least not if a custom resolution has been specified). And the video encoder uses C/C++ standard functions to do the number conversion so it simply stops after the first non-numeric character with no error. And the width works because the argument parser allows specifying option and values as one argument (e.g. `-x1024`).